ansible/roles/invoker/tasks/deploy.yml

```diff
@@ -31,6 +31,24 @@
     linuxOptions: "-v /usr/lib/x86_64-linux-gnu/libapparmor.so.1:/usr/lib/x86_64-linux-gnu/libapparmor.so.1"
   when: whisk_version_name != "mac"
 
+- name: get running invoker information
+  shell: >
+        curl -Gs http://{{ inventory_hostname }}:{{ docker.port }}/containers/json --data-urlencode 'filters={"name":[ "invoker" ]}'
+  register: results
```

Comment:
Couldn't you use the `uri` module as used above to determine the docker root dir?